### PR TITLE
Exit early if no changes

### LINKED ISSUE #25: Fails when there's nothing to do
image

It should probably exit with NO error instead

### PR DESCRIPTION
**Pull request by jcbhmr**
this pr would...
- exit early if no changes to commit
- not push a no-op repo
- not error when no changes

fixes #25

### CODE REVIEW

**Comment by jcbhmr** [comment]
need #58 to be fixed for the test to show green
image

**Comment by jcbhmr** [comment]
@spenserblack ready for review

File: src/index.sh [hunk 1]
```diff
@@ -22,14 +22,21 @@ git remote add origin "https://$INPUT_TOKEN@$INPUT_REPOSITORY.wiki.git"
 git fetch origin
 git reset origin/master
 
+# https://stackoverflow.com/a/2659808
+if git diff-index --quiet HEAD --; then
+  echo 'No changes! Will not commit or push.'
```
Comment (spenserblack) [inline src/index.sh:27]:
```suggestion
	echo 'No changes! Will not commit or push.'
```

**Review by spenserblack — COMMENTED** [review]
Good job! I think it makes sense to add an `allow-empty` input, but perhaps that should be saved for a future PR.

**Review by spenserblack — APPROVED** [review]

**Pull request merged**